state-chain/pallets/cf-lending-pools/src/general_lending.rs

```diff
@@ -602,6 +750,10 @@ impl<T: Config> LoanAccount<T> {
 			return Err(Error::<T>::InsufficientCollateral.into());
 		}
 
+		// If the user requests any additional loans, we assume they no longer want to
+		// be in voluntary liquidation mode (if for whatever reason it was active)
+		self.voluntary_liquidation_requested = false;
```

Comment:
Maybe it would be better to error here? 
If the user tries to take out a loan *during* voluntart liquidation, it's probably a mistake.